_content/the-things-stack-v3/migrate-to-v3.md

```diff
@@ -41,7 +43,7 @@ is translated to a V3 uplink payload formatter as:
 ```js
 function decodeUplink(input) {
     var bytes = input.bytes;
-    var port = input.fport;
+    var port = input.fPort;
   return {
```

Comment:
Wrong indentation here it seems

Reply:
I think what suffices here is that the payload format has changed. Link to the new format.

The old format still works in V3, so to keep things as easy as possible here, don't ask to rewrite the payload codecs.